### DISCUSSION TITLE
密码保护文件夹404错误

### DISCUSSION THREAD

**Discussion by Nofated095** [spencerwooo/onedrive-vercel-index#671] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/49985975/171531732-68652532-97a4-4be4-8864-aca26b9c5da6.png)
如图，在设置完 `.password` 文件和 `protectedRoutes` 后出错。

<https://od.nofated.win/api/?path=/Others/%E4%B8%8D%E5%8F%AF%E4%BB%A5%E6%B6%A9%E6%B6%A9%EF%BC%81>

`{"error":"Internal server error."}`

![image](https://user-images.githubusercontent.com/49985975/171532221-8603e81d-2710-4f82-9b7c-9be9bd593f6b.png)
![image](https://user-images.githubusercontent.com/49985975/171532358-b728af00-86bf-4bfa-8a53-c9067ab0cb27.png)

**Comment by myl7** [comment, marked as answer]
密码是纯数字吗？如果是的话需要参考 #638 修复或者不使用纯数字密码

**Reply by Nofated095** [reply]
重新部署刷新缓存后成功
